### PR TITLE
feat(hybridcloud) Add EU region IPs

### PR DESCRIPTION
**Pull request by markstory**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Add the outbound IP addresses in use by the EU region.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/product/security/ip-ranges.mdx [hunk 1]
```diff
@@ -50,13 +50,21 @@ In some circumstances the Hosted Sentry infrastructure might send HTTP requests
 
 Sentry uses the following IP addresses to make outbound requests:
 
+US Region
```
Comment (mrduncan) [inline docs/product/security/ip-ranges.mdx:53]:
```suggestion
US Data Storage Location
```
Feel free to disagree if region feels better here, I don't feel strongly, but we're referring to them as data storage locations in the rest of our user-facing docs

File: docs/product/security/ip-ranges.mdx [hunk 2]
```diff
@@ -50,13 +50,21 @@ In some circumstances the Hosted Sentry infrastructure might send HTTP requests
...
 ```
 35.184.238.160/32
 104.155.159.182/32
 104.155.149.19/32
 130.211.230.102/32
 ```
 
+EU Region
```
Comment (mrduncan) [inline docs/product/security/ip-ranges.mdx:61]:
```suggestion
EU Data Storage Location
```

**Review by mrduncan — APPROVED** [review]

**Review by tonyo — APPROVED** [review]

File: docs/product/security/ip-ranges.mdx [hunk 3]
```diff
@@ -13,10 +13,13 @@ apply to self-hosted or single tenant.
 
 ## Dashboard and API
 
-Sentry's dashboard and API are both served from a single IP address for all web traffic. The IP address is:
+Sentry's dashboard and API are both served from different of domains depending
+on your organization's data storage location. The IP addresses are:
```
Comment (lizokm) [inline docs/product/security/ip-ranges.mdx:17]:
```suggestion
Sentry's dashboard and API are both served from different domains, depending
on your organization's data storage location. The IP addresses are:
```

File: docs/product/security/ip-ranges.mdx [hunk 4]
```diff
@@ -32,31 +35,51 @@ Sentry's apex domain (`sentry.io`) accepts events from the same IP address as th
...
+
 Sentry's legacy ingestion hostname (`app.getsentry.com`) accepts events from a separate IP address:
 
 ```
 34.96.102.34/32
 ```
 
+Organizations in the EU Data Storage Location are not able to ingest events via
+`sentry.io` or `app.getsentry.com`. Instead, you must use
+`o<number>.ingest.de.sentry.io`.
```
Comment (lizokm) [inline docs/product/security/ip-ranges.mdx:59]:
```suggestion
Organizations in the EU Data Storage Location are unable to ingest events via
`sentry.io` or `app.getsentry.com`. You can use
`o<number>.ingest.de.sentry.io` instead.
```

**Review by lizokm — APPROVED** [review]
Just a few small tweaks. Thanks for updating!

File: docs/product/security/ip-ranges.mdx [hunk 5]
```diff
@@ -13,10 +13,13 @@ apply to self-hosted or single tenant.
...
 
-Sentry's dashboard and API are both served from a single IP address for all web traffic. The IP address is:
+Sentry's dashboard and API are both served from different of domains depending
+on your organization's data storage location. The IP addresses are:
 
 ```
-35.186.247.156/32
+sentry.io 35.186.247.156/32
+us.sentry.io 35.186.247.156/32
+de.sentry.io 34.36.122.224/32
```
Comment (tonyo) [inline docs/product/security/ip-ranges.mdx:22]: We have two addresses reserved for `de.sentry.io`, to facilitate maintenance/upgrades in the future:
```suggestion
de.sentry.io 34.36.122.224/32, 34.36.87.148/32
```

**Pull request merged**